index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css" />
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Special+Elite&display=swap" rel="stylesheet">
+      <title>News Summary</title>
+    </head>
+    <body style="background-image: url('./images/newspapers.jpg');">
+      <div class='header'>Headlines</div>
+      <div style='text-align: center' class='search'>
```

Comment:
A minor one here — the convention is to always use double quotes "" for HTML attributes (even though it'll likely work because the browser handles the single quotes syntax too)